datasets/sentinel-3/collection/sentinel-3-synergy-vg1-l2-netcdf/template.json

```diff
@@ -0,0 +1,310 @@
+{
+    "stac_version": "1.0.0",
+    "type": "Collection",
+    "id": "sentinel-3-synergy-vg1-l2-netcdf",
+    "title": "Sentinel-3 1-Day Surface Reflectance and NDVI",
```

Comment:
Should this title (and the `msft:short_description`) mention something about vegetation?

Reply:
See my response to your comment on the VGP collection template.